src/Core/Geometry/StandardAttribNames.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <Core/RaCore.hpp>
+
+namespace Ra {
+namespace Core {
+namespace Geometry {
+
+/// List of standard vertex attributes.
+/// Corresponding standard vertex attribute names are obtained with g_attribName
+/// Information which is in the mesh geometry
+enum MeshAttrib : uint {
+    VERTEX_POSITION,   ///< Vertex positions
+    VERTEX_NORMAL,     ///< Vertex normals
+    VERTEX_TANGENT,    ///< Vertex tangent 1
+    VERTEX_BITANGENT,  ///< Vertex tangent 2
+    VERTEX_TEXCOORD,   ///< U,V  texture coords (last coordinate not used)
+    VERTEX_COLOR,      ///< RGBA color.
+    VERTEX_WEIGHTS,    ///< Skinning weights (not used)
+    VERTEX_WEIGHT_IDX, ///< Associated weight bones
+
+    MAX_DATA
+};
+
+///@{
```

Comment:
Why defining a documentation group ?

Reply:
I don't know what it is. I think it was a mistake. During the copy/paste operation of my ide.